src/sa_web/static/js/views/place-form-view.js

```diff
@@ -122,6 +164,8 @@ var Shareabouts = Shareabouts || {};
           $button = this.$('[name="save-place-btn"]'),
           spinner, $fileInputs;
 
+      model.attributes["from_dynamic_form"] = true;
```

Comment:
I wonder if there is a more elegant way to handle the dynamic form state? Although we can always refactor this after the merge